modules/images-update-global-pull-secret.adoc

```diff
@@ -25,18 +25,43 @@ As of {product-title} 4.7, changes to the global pull secret no longer trigger a
 
 .Prerequisites
 
-* You have a new or modified pull secret file to upload.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+
 .Procedure
+To update the global pull secret, complete the following steps:
+[NOTE]
+====
+To overwrite your existing pull secret using a new pull secret file, skip steps 1 & 2.
```

Comment:
This isn't one of the places where you can use an ampersand in running text. See the "Individual punctuation marks or special characters" section of the IBM Style Guide.

Because the steps are automatically generated, we don't refer to steps by number in text. I think it would be best to remove this note and present the first two steps as optional substeps with the reason why you'd do it, using something like this outline:

~~~
. Optional: To overwrite your existing pull secret using a new pull secret file, complete the following steps:
.. Download the pull secret:

.. Add the new pull secret:

. Update the global pull secret for your cluster:
```